src/main/java/com/project/psk/model/entity/GuestEntity.java

```diff
@@ -0,0 +1,49 @@
+package com.project.psk.model.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class GuestEntity {
+    private UUID id;
+    private String name;
+    private String email;
+    private UUID eventId;
+
+    public UUID getId() {
+        return id;
+    }
+
+    public void setId(UUID id) {
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public UUID getEventId() {
+        return eventId;
+    }
+
+    public void setEventId(UUID eventId) {
+        this.eventId = eventId;
+    }
```

Comment:
I don't think you need to write any of these, as @Data annotation generates these automatically: "@Data is a convenient shortcut annotation that bundles the features of @ToString, @Getter / @Setter and @RequiredArgsConstructor together"